src/python/grpcio/grpc/_channel.py

```diff
@@ -926,6 +926,11 @@ def __init__(self, target, options, credentials):
         self._call_state = _ChannelCallState(self._channel)
         self._connectivity_state = _ChannelConnectivityState(self._channel)
 
+        # Temporary work around UNAVAILABLE issues
```

Comment:
I don't think this change is sufficient to close #11043 because #11043 is a wrapped-languages issue and this is a Python-only change.

#9884 might be the right issue for scoping the temporary-ness of this workaround; I think I'm a little thrown by the way it's being called "retries" and being described as a feature when I keep encountering it as "not trying hard enough the first time" and thinking of it as a bug.